### PR TITLE
Added CTR Acceptance Tests

### LINKED ISSUE #208: Add test for CTR task application parameters
need to be able to verify that passing parameters to CTR works as expected.   i.e. `task create my-composed-task --definition "mytaskapp --displayMessage=hello && timestamp --format=YYYY"`

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #208

Set Base Task App Version to Elston

Updated the readme

### CODE REVIEW

File: acceptance-tests/spring-cloud-dataflow-acceptance-tests/src/test/java/org/springframework/cloud/dataflow/acceptance/tests/AbstractDataflowTests.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
 public abstract class AbstractDataflowTests {
 
 	private final static String STREAM_APPS_URI = "https://repo.spring.io/libs-release-local/org/springframework/cloud/stream/app/spring-cloud-stream-app-descriptor/Celsius.SR3/spring-cloud-stream-app-descriptor-Celsius.SR3.stream-apps-rabbit-maven";
-	private final static String TASK_APPS_URI = "https://repo.spring.io/libs-release/org/springframework/cloud/task/app/spring-cloud-task-app-descriptor/Clark.SR1/spring-cloud-task-app-descriptor-Clark.SR1.task-apps-maven";
```
Comment (sabbyanandan) [inline acceptance-tests/spring-cloud-dataflow-acceptance-tests/src/test/java/org/springframework/cloud/dataflow/acceptance/tests/AbstractDataflowTests.java]: Can we instead use the dataflow.spring.io redirects, please? Specifically, the `latest` links? That way we don't have to deal with updating this whenever a new release is done. We will automatically be on the latest GA release of the Apps.

When you change it, please also change the stream apps. All the latest links can be found in the App Starters project sites (look for the table at the bottom).

If we agree with this suggestion, we can knock-out https://github.com/spring-cloud/spring-cloud-dataflow-acceptance-tests/issues/205, too, on this PR!!

Comment (cppwfs) [reply acceptance-tests/spring-cloud-dataflow-acceptance-tests/src/test/java/org/springframework/cloud/dataflow/acceptance/tests/AbstractDataflowTests.java]: 👍

Comment (ilayaperumalg) [reply acceptance-tests/spring-cloud-dataflow-acceptance-tests/src/test/java/org/springframework/cloud/dataflow/acceptance/tests/AbstractDataflowTests.java]: Looks like the number of the environment variables that need to be changed are more and hence, I will have the changes to use the `dataflow.spring.io` redirects as a separate PR after merging this PR.

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Pull request merged**